### ISSUE TITLE
Nan Loss

### ISSUE THREAD

**Issue by shidilrzf** [hwalsuklee/tensorflow-mnist-VAE#4]
Sometimes when I try this code, I get a Nan loss. For example the loss for epoch "i" is 100 and the loss for epoch "i+1" is Nan...
Do you have any idea why this happens?

**Comment by jaekyeom** [comment]
That's probably because `tf.log` got `0` as input. Try using `tf.nn.sigmoid_cross_entropy_with_logits`.
